bgpd/valgrind.supp

```diff
@@ -15,3 +15,18 @@
    fun:qcalloc
    fun:zlog_target_clone
 }
+{
+   <libyang1_1.0.184>
+   Memcheck:Leak
+   match-leak-kinds: reachable
+   fun:calloc
+   fun:_dlerror_run
+   fun:dlopen@@GLIBC_2.2.5
+   obj:/usr/lib/x86_64-linux-gnu/libyang.so.1.9.2
+   fun:ly_load_plugins
```

Comment:
nit: I think it'd be good to remove all lines below this one (please see commit d09b51d12775).

Also, please remove the entry for libyang_0.16.46 which we don't support anymore.